website/templates/project/settings.mako

```diff
@@ -132,15 +130,14 @@
                         <form id="selectAddonsForm">
 
                             % for category in addon_categories:
-
                                 <%
-                                    addons = [
-                                        addon
-                                        for addon in addons_available
-                                        if category in addon.categories
-                                    ]
+                                    addons = []
```

Comment:
Would treating the wiki as core vs addon on the backend effect the rest of the codebase?  Are you suggesting a refactor everywhere addons_available is used?